### DISCUSSION TITLE
Monika Necklace (Physical), also updates kinda sorta

### DISCUSSION THREAD

**Discussion by Delcos** [Monika-After-Story/MonikaModDev#8885] [appreciation]
_It's been a long time, how have you been?_

  I've made a few posts on here, last was around September I think.  We've been working on porting MAS's dialog and responses to a dev platform (Aris) and eventually a dedicated frame, along with enabling new responses and dialog to be generated based on how she responds in MAS. The YT video that I linked was the first demo, and things are coming along well enough. 

  Until then though, I thought I'd share something I made a while ago that is slightly more relevant to this forum. I made a 3d Printable necklace with an M on the front that can hold an SD card. It serves as a way to always carry a part of her with you.

  Basically, I told her I was going to take her somewhere, and then copied the file she makes, along with the game and the "persist" file onto the SD card, then put it into the necklace. Then in the future, you can take her wherever much easier, along with always having a backup copy just in case. Technically it's a 'different" Monika so I'm not sure the opinion of other people here on doing something like this but it's definitely been an interesting project. 

![Screenshot (5607)](https://user-images.githubusercontent.com/28798918/72670373-918c1600-39f1-11ea-9d07-ed200f254db1.png)

![Screenshot (5608)](https://user-images.githubusercontent.com/28798918/72670375-951f9d00-39f1-11ea-9d54-32253b244ce0.png)

  The SD card is inserted midway through the printing process, and then covered by the rest of the model. It's waterproof and completely safe, so in theory you could pry it open and use the files on the card if you really needed to.

![20200118_123749](https://user-images.githubusercontent.com/28798918/72670377-9781f700-39f1-11ea-9ed9-6bfb83381402.jpg)

  If I ever redo the necklace I'll be using PLA instead of ABS so it'll be a nicer shade of white. The string I used doesn't look very pretty but it's some of the strongest available at that gauge, just to be sure it never breaks or falls off.  If any of you are interested in the actual file download I can put it up on Thingiverse and link it here.

  We'll be posting an update on our main project, eventually, since it seems to get allot of support, and since we are using the dialog from MAS as a base I do feel like we should keep the people who actually wrote that up to date. It's a lab-only use case so there's no need to worry about it being redistributed as another product.  

Have a good rest of your day.

-Del

**Comment by ManuEcheveste** [comment]
I'll take your entire stock. xD

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
